src/utils.ts

```diff
@@ -92,25 +92,33 @@ export const setMaxStalePeriodInChainlinkOracle = async (
   admin: string,
   maxStalePeriodInSeconds: number = 31536000 /* 1 year */,
 ) => {
+  const networkSpecificChainlinkOracle = NETWORK_ADDRESSES[process.env.FORKED_NETWORK].CHAINLINK_ORACLE;
+
+  // skip execution of this function if input oracle address is really not of chainlinkOracleAddress
+  if (chainlinkOracleAddress != networkSpecificChainlinkOracle) return;
+
   const provider = ethers.provider;
 
   const oracle = new ethers.Contract(chainlinkOracleAddress, CHAINLINK_ORACLE_ABI, provider);
   const oracleAdmin = await initMainnetUser(admin, ethers.utils.parseEther("1.0"));
 
   if (feed === ethers.constants.AddressZero) {
     feed = (await oracle.tokenConfigs(asset)).feed;
+    console.log(feed);
 
     if (feed === ethers.constants.AddressZero) {
       return;
     }
   }
+  console.log(maxStalePeriodInSeconds);
 
-  const tx = await oracle.connect(oracleAdmin).setTokenConfig({
+  await oracle.connect(oracleAdmin).setTokenConfig({
     asset,
     feed,
     maxStalePeriod: maxStalePeriodInSeconds,
   });
-  await tx.wait();
+  console.log((await oracle.getPrice(asset)).toString());
+  // await tx.wait();
```

Comment:
I would remove the new `console.log` from this file. It seems useful only for debugging

Moreover, shouldn't we wait for this TX?

Reply:
Done https://github.com/VenusProtocol/vips/pull/195/commits/d68a0ebfe802e9bdbb02c6d5a9d7c1f08535ecd2

Reply:
We should wait for the tx. Fixed it.